src/Mappers/InvoiceMapper.php

```diff
@@ -111,6 +111,35 @@ public static function map(Response $response)
             $invoice->addLine($invoiceLine);
         }
 
+        // Financial elements and their methods
+        $financialsTags = array(
+            'code'      => 'setFinancialCode',
+            'number'    => 'setFinancialNumber'
+        );
+
+        // Financial elements
+        $financialElement = $responseDOM->getElementsByTagName('financials')->item(0);
+
+        if ($financialElement !== null) {
+
+            // Go through each financial element and add to the assigned method
+            foreach ($financialsTags as $tag => $method) {
+
+                // Get the dom element
+                $_tag = $financialElement->getElementsByTagName($tag)->item(0);
+
+                // If it has a value, set it to the associated method
+                if (isset($_tag) && isset($_tag->textContent)) {
```

Comment:
Could you try to use the methods in `\PhpTwinfield\Mappers\BaseMapper`?

Reply:
I copied those from the other `Mappers` that have the `financials` implemented, but I've fixed it in this one :+1:.